src/Core/Hook/HookDispatcherAwareTrait.php

```diff
@@ -43,6 +45,8 @@ trait HookDispatcherAwareTrait
      */
     public function setHookDispatcher(HookDispatcherInterface $hookDispatcher)
     {
+        @trigger_error('HookDispatcherAwareTrait is deprecated as of 1.7.5 and will be removed in the next major version. If you need to inject HookDispatcherInterface use the constructor not setter injection.', E_USER_DEPRECATED);
```

Comment:
```suggestion
        @trigger_error('HookDispatcherAwareTrait is deprecated as of 1.7.5.1 and will be removed in the next major version. If you need to inject HookDispatcherInterface, use the constructor instead of setter injection.', E_USER_DEPRECATED);
```

Reply:
done